bodhi/tests/server/scripts/test_approve_testing.py

```diff
@@ -153,7 +156,42 @@ def test_exception_handler(self, stdout, exit, remove, comment):
         comment.assert_called_once_with(
             self.db,
             ('This update can be pushed to stable now if the maintainer wishes'),
-            author='bodhi')
+            author='bodhi',
+            email_notification=True,
+        )
+        self.assertEqual(stdout.getvalue(),
+                         f'{update.alias} now meets testing requirements\nThe DB died lol\n')
+        remove.assert_called_once_with()
+
+    @patch('bodhi.server.models.Update.comment', side_effect=IOError('The DB died lol'))
+    @patch('bodhi.server.scripts.approve_testing.Session.remove')
+    @patch('bodhi.server.scripts.approve_testing.sys.exit')
+    @patch('sys.stdout', new_callable=StringIO)
+    def test_exception_handler_no_email(self, stdout, exit, remove, comment):
+        """The Exception handler prints the Exception, rolls back and closes the db, and exits."""
```

Comment:
Please explain in the docstring why this test is different from the previous one.
It would be ideal if the similar lines could be factored.

Reply:
docstring adjusted :)